### ISSUE TITLE
Load charts that you need

### ISSUE THREAD

**Issue by mato75** [PMSI-AlignAlytics/dimple#172]
Is there a possibility to load charts/js that we want to use, without requirejs?

Some sort of build, where you select which charts you want in minified JS file.

**Comment by johnkiernander** [comment]
Yes, although Dimple supports requirejs there is no requirement for it.  In fact, we don't use it in our projects.

**Comment by mato75** [comment]
How? Can you give me an example or a suggestion, where I can do this?

**Comment by johnkiernander** [comment]
Maybe I'm not entirely clear with what you are trying to achieve but the examples use dimple without requirejs.  Any example (e.g. https://github.com/PMSI-AlignAlytics/dimple/blob/master/examples/bars_vertical.html) just references dimple using a url and though the chart code is directly in a script tag, it could equally be in an external JavaScript file.

**Comment by mato75** [comment]
Aha, ok no that is not what I am asking.

I would like to know, how do I have to change Gruntfile build, so that it only includes js files for types of charts that I will use. I do not the whole js file with all chart types, if I don't have them on my page.
